applications/mne_analyze/plugins/averaging/averaging.pro

```diff
@@ -0,0 +1,149 @@
+#==============================================================================================================
+#
+# @file     averaging.pro
+# @author   Lorenz Esch <lesch@mgh.harvard.edu>;
+#           Gabriel Motta <gbmotta@mgh.harvard.edu>
+# @since    0.1.0
+# @date     Februrary, 2020
+#
+# @section  LICENSE
+#
+# Copyright (C) 2020, Lorenz Esch, Gabriel Motta. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification, are permitted provided that
+# the following conditions are met:
+#     * Redistributions of source code must retain the above copyright notice, this list of conditions and the
+#       following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and
+#       the following disclaimer in the documentation and/or other materials provided with the distribution.
+#     * Neither the name of MNE-CPP authors nor the names of its contributors may be used
+#       to endorse or promote products derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED
+# WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+# PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+#
+# @brief    This project file generates the makefile for the averaging plugin.
+#
+#==============================================================================================================
+
+include(../../../../mne-cpp.pri)
```

Comment:
make sure the averaging is added [here](https://github.com/mne-tools/mne-cpp/blob/baf467779652b5a1ad63c76d87955110d087a526/applications/mne_analyze/mne_analyze/mne_analyze.pro#L83) and [here](https://github.com/mne-tools/mne-cpp/blob/baf467779652b5a1ad63c76d87955110d087a526/applications/mne_analyze/mne_analyze/main.cpp#L75) in order to make them accessible in the static and wasm version.